products/sle15/profiles/cis.profile

```diff
@@ -1,970 +1,22 @@
 documentation_complete: true
 
 metadata:
-    version: 1.0.0
+    version: 2.0.0
```

Comment:
```suggestion
    version: 1.0.1
```

The version here is intended to be the same as in the policy. Please change in other cis profiles files as well.

Reply:
Done.